### DISCUSSION TITLE
(Looking for Help) Godot 4+ Multiplayer Seamless Open-World Chunks

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [godotengine/godot-proposals#9109] [General Discussions]
https://jonathaneeckhout.itch.io/jdungeon For the goal right now it is easy to play this but imagine it has seamless chunk loading and unloading. As instead of the black world boundary a new chunk loads in seamlessly. What are your suggestions and thoughts?

Update: https://github.com/WithinAmnesia/ARPG/tree/ARPG-Infinite-Worlds Here is the 128x128 with 32x32 pixel tile chunk to test.

My initial testing seems to feel even faster combat with the 128x128 chunk. This 128x128 chunk has the same amount of entities as the 256x256 chunk. For I moved all of the entities over into the 128x128 chunk. This 128x128 chunk is a good test for using the Infinite-Worlds using the BinarySerializer for Godot 4.2+ from Theraot: https://gist.github.com/theraot/31515e28e2d8bfea33f6c6d5bcd852f6 . There needs to be some testing how to make the chunks seamlessly load and unload. https://gamedev.stackexchange.com/questions/209002/looking-for-help-godot-4-multiplayer-seamless-open-world-chunks

I'm trying to find a way to seamless load and unload chunks for a 2D multiplayer game project to make an open world with a working server using Godot 4.2.1.NET. Example: https://www.youtube.com/watch?v=uE36MVjB5-8

How can this work for multiplayer and what is needed for this to potentially work? What options can be used for chunk loading and unloading seamlessly in Godot 4.2.1.NET? Please give feedback. Any help with the main proposal would help: https://github.com/godotengine/godot-proposals/issues/9108

Test chunks link: https://github.com/WithinAmnesia/ARPG/discussions/15

**Comment by octanejohn** [comment]
ask in the forum https://forum.godotengine.org/

**Reply by WithinAmnesia** [reply]
https://forum.godotengine.org/t/looking-for-help-godot-4-multiplayer-seamless-open-world-chunks/47902/2 I posted this question to the Godot forums. Thank you for your suggestion.

**Comment by WithinAmnesia** [comment]
https://github.com/Zylann/voxelgame/issues/101#issuecomment-1958792464 Big 2D/3D hybrid progress update!
